charts/nd-common/templates/_monitors.tpl

```diff
@@ -52,7 +52,7 @@ spec:
     matchLabels:
       {{- include "nd-common.selectorLabels" . | nindent 6 }}
   podMetricsEndpoints:
-    - port: {{ .Values.monitor.portName }}
+    - port: {{ .Values.monitor.portNumber }}
```

Comment:
According to the spec, this is supposed to be the name of the port: https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#podmetricsendpoint

What is prompting you to try to change this?